lmi_utils/label_utils/csv_to_yolo.py

```diff
@@ -202,7 +202,8 @@ def convert_to_txt(fname_to_shapes, target_classes, is_seg=False, is_convert=Fal
                     new_rows[key].extend(kp)
             if not hit:
                 raise Exception(f'key point ({x},{y}) is not in any bbox. Fix it.')
-                    
+        if len(new_rows) == 0 and len(rows) == 0:
+            continue
```

Comment:
The users can turn on the ''bg" flag through the terminal command. We also need to support output background images.